src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -446,6 +446,14 @@ class Typer extends Namer with TypeAssigner with Applications with Implicits wit
           case _ =>
         }
         checkClassType(tpt1.tpe, tpt1.pos, traitReq = false, stablePrefixReq = true)
+
+        tpt1 match {
+          case AppliedTypeTree(_, targs) =>
+            for (targ @ TypeBoundsTree(_, _) <- targs)
+              ctx.error("type argument must be fully defined", targ.pos)
```

Comment:
Agreed in principle, but I think it's best to leave that to others who are already in it. Crafting a good error message takes time. But my goal here is to clear the backlog of fuzzing errors, so I want to move quickly.